### DISCUSSION TITLE
Textured Vertex

### DISCUSSION THREAD

**Discussion by edwinsyarief** [hajimehoshi/ebiten#1534] [Q&A] [answered]
Hi @hajimehoshi ,

Is it possible to render line with texture?

I'm trying to render dynamic rope with texture

Thank you.

**Comment by hajimehoshi** [comment, marked as answer]
It is theoretically possible with `DrawTriangles`, but this requires some calculations.

**Reply by edwinsyarief** [reply]
I see, I just want to know how to apply texture.

For your example polygon. Maybe can you give me suggestion how to apply texture, what should I do, what func should I call to set texture.

Thank you.

**Reply by hajimehoshi** [reply]
You can specify the texture at `img` argument at [`DrawTriangles`](https://pkg.go.dev/github.com/hajimehoshi/ebiten/v2#Image.DrawTriangles). You need to calculate appropriate source and destination vertices of [`Vertex`](https://pkg.go.dev/github.com/hajimehoshi/ebiten/v2#Vertex) struct values. Ebiten provides an low level API and doesn't have support rendering a texture rope directly.

**Reply by edwinsyarief** [reply]
Thank you. That's all I want to know.
